cmd/show/constants.go

```diff
@@ -0,0 +1,63 @@
+package show
+
+import (
+	"github.com/charmbracelet/bubbles/key"
+	"github.com/charmbracelet/bubbles/list"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+)
+
+// WindowSize stores the size of the terminal
+var WindowSize tea.WindowSizeMsg
+
+// Keymaps
+var OpenPR = key.NewBinding(key.WithKeys("O"), key.WithHelp("O", "open pr"))
```

Comment:
I ran into an interesting bug (we can fix this in a future patch if we want) where filtering on something that has a "O" opens the PR immediately. So a filter like:

```
/CODEOWNER
```
will open the first pr as soon as the first "O" is entered.

Reply:
Should now be fixed on the latest commit. On any ```tea.KeyMsg``` will now checks the internal ```filterState``` of the list to prevent the keymap collision.